src/transport_ofi.c

```diff
@@ -90,15 +90,13 @@ shmem_internal_mutex_t          shmem_transport_ofi_lock;
 
 struct fabric_info shmem_transport_ofi_info = {0};
 
-static shmem_transport_ctx_t** shmem_transport_ofi_contexts;
-static size_t shmem_transport_ofi_num_ctx = 0;
+static shmem_transport_ctx_t** shmem_transport_ofi_contexts = NULL;
+static size_t shmem_transport_ofi_contexts_size = 0;
```

Comment:
To me, `num_ctx` makes slightly more sense than `contexts_size`, but it's no big deal.   If it a namespace sorta thing, maybe `contexts_num`?

Reply:
"contexts" is the array name and "contexts_size" is the size of the array.  We no longer track the number of context references stored in the array (since we don't use it anywhere, it was unnecessary global state).  So, "contexts_size" seemed clearer than "contexts_num", but I'm happy to rename either or both if you have a better name in mind.

Reply:
Yeah, "contexts_size" is good.  I just tend to first think of "sizeof" and "size_t", and tend towards "num" and "len" for number of elements, but that's personal preference.  No problem either way, just a thought.

This PR looks great, BTW, it's much cleaner.  Ready to merge when you are.

Reply:
Oh, "len" is good.